sematic/ui/packages/common/src/component/menu.tsx

```diff
@@ -0,0 +1,29 @@
+import Grid from "@mui/material/Grid";
+import Button from "@mui/material/Button";
+import Box from "@mui/material/Box";
+import useTheme from "@mui/material/styles/useTheme";
+import Fox from "src/static/fox";
+
+const HeaderMenu = () => {
+    const theme = useTheme();
+
+    return <Grid container spacing={0}>
```

Comment:
The menu has a bottom border.

Reply:
Thanks for noticing it, added the bottom border now.